pgml-dashboard/src/api/cms.rs

```diff
@@ -26,6 +26,7 @@ lazy_static! {
         "Blog",
         true,
         HashMap::from([
+            ("the-1.0-sdk-is-here", "the-1.0-sdk-is-here"),
```

Comment:
You only need this if you're mapping one url to another to fix renamed posts. This looks identical?